eng/Versions.props

```diff
@@ -8,7 +8,7 @@
     Roslyn version
   -->
   <PropertyGroup>
-    <VersionPrefix>3.1.0</VersionPrefix>
```

Comment:
It'll be important to make sure we don't mess up the version number when we flow this into 16.2, etc.